dir.targets

```diff
@@ -45,4 +45,15 @@
     <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
     <PortableNuGetMode>true</PortableNuGetMode>
   </PropertyGroup>
+
+  <!--
+    add the test-runtime project.json to the list of supplemental test data so it will get included
+    in the test archive; the stress test build system depends on it when building test mixes.
+  -->
+  <ItemGroup Condition="'$(IsTestProject)'=='true'">
```

Comment:
@weshaggard @jhendrixMSFT we need this because tests in the CoreFX repo all reference this project.json as a project reference.  Therefore in order to enumerate all there dependencies when we are generating stress tests we need to be able to include this file as well.  As far as I have seen this is unique to CoreFX, but nothing precludes other repo's tests from doing this.  It's possible that other repos will have similar requirements later on, but since that isn't the case now I think it would be hard to determine how to best generalize it.